src/Microsoft.Health.SqlServer/Features/Schema/SchemaInitializer.cs

```diff
@@ -268,6 +264,8 @@ public static void ValidateDatabaseName(string databaseName)
         {
             throw new InvalidOperationException("The initial catalog in the connection string cannot be a system database");
         }
+
+        return databaseName;
```

Comment:
What is the purpose of returning the database that is passed as an argument?

Reply:
- To assign the result after valuation. Just for convenience